CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs then need to be appropriately assigned and labelled by someone with the appropriate permissions.
+
+1. Fork this repository.
+2. Clone your fork of the repository.
+3. When developing, keep you changes as atomic as possible.
+4. When you are ready to make a PR, fetch from upstream to make sure you have the latest changes from the upstream repository. Add your commit on top of any new commits from upstream:
+
+```
+git fetch origin
+git rebase origin/develop
+git checkout <your branch>
+git rebase develop
+```
+
+5. Set your local branch to track its equivalent on the remote server. Push your changes to that remote branch.
+6. Create a PR.
+
+# PR Review Instructions for Internal Members
+
+1. Add the appropriate label to the PR. If you do not have the ability to do so, reach out to Marcin Wadon (marcin.wadon@gmail.com).
```

Comment:
If this will not be removed, please take a look at Ryle's comment related to that part

Reply:
The section has been removed.